test/e2e/lib/pages/gutenboarding/acquire-intent-page.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { By } from 'selenium-webdriver';
+
+/**
+ * Internal dependencies
+ */
+import AsyncBaseContainer from '../../async-base-container';
+import * as driverHelper from '../../driver-helper';
+
+export default class AcquireIntentPage extends AsyncBaseContainer {
+	constructor( driver ) {
+		super( driver, By.css( '.acquire-intent' ) );
+	}
+
+	async enterSiteTitle( siteTitle ) {
+		const siteTitleSelector = By.css( '#site-title__input' );
+		return await driverHelper.setWhenSettable( this.driver, siteTitleSelector, siteTitle );
+	}
+
+	async goToNextStep() {
+		const nextButtonSelector = By.css( '.acquire-intent__next' );
+		return await driverHelper.clickWhenClickable( this.driver, nextButtonSelector );
+	}
+
+	async skipStep() {
+		const skipButtonSelector = By.css(
+			this.screenSize === 'MOBILE'
+				? '.acquire-intent__footer .acquire-intent__skip-site-title'
```

Comment:
`acquire-intent__footer` still there?

Reply:
Yes, it's just in a different position. 
We need separate HTML for desktop/mobile because of the layout: skip button on desktop is relative to the input field. I feel this would change soon so we'll update also the selector here 🤓